src/platform/Zephyr/BLEManagerImpl.cpp

```diff
@@ -423,6 +464,9 @@ CHIP_ERROR BLEManagerImpl::_SetAdvertisingMode(BLEAdvertisingMode mode)
     case BLEAdvertisingMode::kSlowAdvertising:
         mFlags.Set(Flags::kFastAdvertisingEnabled, false);
         break;
+    case BLEAdvertisingMode::kExtendedAdvertising:
+        mFlags.Set(Flags::kExtendedAdvertisingEnabled, true);
```

Comment:
Apart from clearing `kExtendedAdvertisingEnabled` somewhere, should we clear other flags in this `case`?